amf-client/shared/src/test/resources/org/raml/parser/include/include-jsonschema-typedeclaration-refs/output.txt

```diff
@@ -3,43 +3,41 @@ title: Staff API
 version: v1-dev
 types:
   User:
-    type: |-
-      {
-        "$schema": "http://json-schema.org/draft-04/schema#",
-        "title": "Example Schema",
-        "type": "object",
-        "properties": {
-          "firstName": {
-            "type": "string"
-          },
-          "lastName": {
-            "type": "string"
-          },
-          "age": {
-            "description": "Age in years",
-            "type": "integer",
-            "minimum": 0
-          },
-          "friends": {
-            "description": "A list of friends",
-            "type": "array",
-            "items": {
-              "$ref": "friends.json#/definitions/person"
-            }
-          }
-        },
-        "required": [
-          "firstName",
-          "lastName",
-          "friends"
-        ]
-      }
+    displayName: Example Schema
```

Comment:
Undesired change but I don't think it is relevant. It appears to lose annotations in inheritance resolving in the ShapeCanonizer. Debugging it is a mess.

We should discuss wether this is relevant or not.

Reply:
Haven't a json schema emitter after resolution makes perfect sense to me.
After resolve the shape, is not the same any more. In other cases, the inheritance could be lost. (Same happen here).